terraform/root.tf

```diff
@@ -0,0 +1,50 @@
+data "aws_ssm_parameter" "cost_centre" {
+  name = "/mgmt/cost_centre"
+}
+
+locals {
+  environment = "mgmt"
+  aws_region  = var.default_aws_region
+  common_tags = tomap(
+  {
+    "Environment"     = local.environment,
+    "Owner"           = "TDR",
+    "Terraform"       = true,
+    "TerraformSource" = "https://github.com/nationalarchives/tdr-jenkins/tree/master/terraform",
+    "CostCentre"      = data.aws_ssm_parameter.cost_centre.value
+  }
+  )
+}
+
+provider "aws" {
+  region = local.aws_region
+}
+
+variable "default_aws_region" {
+  default = "eu-west-2"
+}
+
+terraform {
```

Comment:
Can you split the `terraform {... }` from line 27 to 42 into a versions.tf file, just to keep it consistent with the Terraform structure in all the other repos?